admin/actions.php

```diff
@@ -696,22 +706,28 @@ function hmbkp_ajax_cron_test() {
 
 	}
 
-	$url = site_url( 'wp-cron.php' );
+	$spawn1 = hmbkp_get_cron_spawn();
+	$spawn2 = hmbkp_get_cron_spawn();
+	$spawn3 = hmbkp_get_cron_spawn();
+
+	$code1    = wp_remote_retrieve_response_code( $spawn1 );
```

Comment:
Why switch to calling `wp_remote_retrieve_response_code ` here rather than in the `array_map` on https://github.com/humanmade/backupwordpress/pull/822/files#diff-b2d8a71a75e0cf40115b3529a31e0761L712